virtualizarr/readers/kerchunk.py

```diff
@@ -38,7 +38,7 @@ def open_virtual_dataset(
         fs = _FsspecFSFromFilepath(filepath=filepath, reader_options=reader_options)
 
         # The kerchunk .parquet storage format isn't actually a parquet, but a directory that contains named parquets for each group/variable.
-        if fs.filepath.endswith("ref.parquet"):
+        if fs.filepath.endswith(".parquet"):
```

Comment:
according to https://github.com/fsspec/kerchunk/issues/519#issuecomment-2429763457, you could also check for the presence of `f"{fs.filepath}/.zmetadata"`, which should avoid trying to open actual parquet files – the only downside is a additional request, but since we already check magic bytes elsewhere I don't think that's a concern:
```suggestion
        if fs.filepath.endswith(".parquet") and fs.fs.isfile(f"{fs.filepath}/.zmetadata"):
```
(or define `isfile` / `exists` on `_FsspecFSFromFilepath` and use that instead of `fs.fs.isfile`)

cc @martindurant

Reply:
Thanks @keewis! 

> Also, do we want to support .parq, as well, or do we expect people to always end the filename with .parquet? If we do want to support .parq, you can use fs.filepath.endswith((".parquet", ".parq")) (not that I'd need that myself).

I don't have really strong opinions on this. Adding more file name guessing doesn't feel great to me. I could see people naming files with a `.pqt` suffix since it's really just a directory name we're looking at. Maybe we just stick with `.parquet` and mention in the docs and/or in the ValueError?

cc @TomNicholas

Reply:
fine with me!

Reply:
I don't have much of an opinion. Explicit is better than guessing, but clear documentation and error message either way.

Reply:
Thanks both! I added a bit to the error message.